### ISSUE TITLE
`table-cell-padding` reports empty cells

### ISSUE THREAD

**Issue by jchaps** [wooorm/remark-lint#104]
```
| A   |     |
| --- | --- |
| abc | def |
```

I'm using some blank cells in markdown tables, and remark-lint is reporting that the cell isn't padded.  It's actually over-padded since there is no valid string in the cell.

A suggestion:
If style= "padded", accept 3 spaces between pipes.
If style = "compact", accept 1 space between pipes.

**Comment by jchaps** [comment]
Thank you for the quick fix!

**Comment by wooorm** [comment]
Not too quick 😉 
I’ll try doing some more issues, and release a bit later. Hopefully today.

Thanks for the issue!
